tests/ottest/fixtures/blockchain/RegtestSimple.cpp

```diff
@@ -469,11 +473,60 @@ void Regtest_fixture_simple::SendCoins(
     std::cout << sender.name_ + " send " << coins_to_send << " to "
               << receiver.name_ << " address: " << address << std::endl;
 
+    const auto& network_receiver =
+        receiver.api_->Network().Blockchain().GetChain(test_chain_);
+    auto output_size =
+        network_receiver.Wallet()
+            .GetOutputs(
+                receiver.nym_->ID(), blockchain::node::TxoState::ConfirmedNew)
+            .size();
     auto future = network.SendToAddress(
         sender.nym_id_, address, coins_to_send, memo_outgoing);
 
     // We need to confirm send transaction, so it is available after restore
     MineTransaction(sender, future.get().second, current_height);
+    WaitForOutputs(receiver, send_transactions_, output_size);
```

Comment:
Mine is appending send_transactions_ and we send it nextly by param to WaitForOutputs, if it would not make big overhead it would be good to make it local, to achieve clearer code

Reply:
We use it globally in different test e.x. StressRestartAtEnd